lib/elixir/test/elixir/registry_test.exs

```diff
@@ -602,7 +602,7 @@ defmodule RegistryTest do
   end
 
   test "raises if :name is missing" do
-    assert_raise ArgumentError, ~r/expected :name/, fn ->
+    assert_raise ArgumentError, ~r/expected :name option to be present/, fn ->
```

Comment:
Can we quickly add a test for the case where `:name` is present but not an atom? Then we're good to merge!